osh/hub/settings.py

```diff
@@ -19,6 +19,9 @@
 # file to read the real SECRET_KEY from
 SECRET_KEY_FILE = "/var/lib/osh/hub/secret_key"
 
+# where to read API keys from
+SETTINGS_DIR = "/etc/osh/hub/secrets"
```

Comment:
I would name it `SECRETS_DIR` or `SECRET_SETTINGS_DIR` to distinguish it from other directories that contain configuration files.